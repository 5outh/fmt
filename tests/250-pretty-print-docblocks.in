<?php
//passes:PrettyPrintDocBlocks,Default

/**
 * message
 * more message
 *
 * @param type2 $a descriptionáccented
 * @return array
 * @param LongType $b description2
 */
function a(type2 $a, LongType $b) {

}