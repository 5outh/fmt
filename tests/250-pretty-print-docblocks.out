<?php
//passes:PrettyPrintDocBlocks,Default

/**
 * message
 * more message
 *
 * @param  type2    $a descriptionáccented
 * @param  LongType $b description2
 * @return array
 */
function a(type2 $a, LongType $b) {

}